scm/scm.go

```diff
@@ -18,14 +18,10 @@ type SCM interface {
 	CreateRepository(context.Context, *CreateRepositoryOptions) (*Repository, error)
 	// Get repositories within organization.
 	GetRepositories(context.Context, *qf.Organization) ([]*Repository, error)
-	// Delete repository.
-	DeleteRepository(context.Context, *RepositoryOptions) error
 	// Returns true if there are no commits in the given repository
 	RepositoryIsEmpty(context.Context, *RepositoryOptions) bool
 	// Create team.
-	CreateTeam(context.Context, *NewTeamOptions) (*Team, error)
-	// Delete team.
-	DeleteTeam(context.Context, *TeamOptions) error
+	CreateTeam(context.Context, *TeamOptions) (*Team, error)
```

Comment:
Do we still need this separate from CreateGroup?

Reply:
I'm attempting to remove `CreateTeam` and `CreateRepository` in another PR.